FlipperZero/FlipperZero.xcodeproj/project.pbxproj

```diff
@@ -103,6 +103,7 @@
 		498305C22527FEE900B04AFD /* Born2bSportyV2.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; path = Born2bSportyV2.ttf; sourceTree = "<group>"; };
 		49BE36C6253BE71200F727EF /* .xcccr.toml */ = {isa = PBXFileReference; lastKnownFileType = text; path = .xcccr.toml; sourceTree = "<group>"; };
 		49CE8F9D25262E2300B9CBE4 /* LaunchScreen.storyboard */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; path = LaunchScreen.storyboard; sourceTree = "<group>"; };
+		498305F62529569200B04AFD /* .swiftlint.yml */ = {isa = PBXFileReference; lastKnownFileType = text.yaml; path = .swiftlint.yml; sourceTree = "<group>"; };
```

Comment:
Is this a duplicate of line 101?

Reply:
Yeah, my bad, accepted on squash. Anyway it isn’t an issue for Xcode and will be automatically eliminated on any further file operation (e.g. reordering, new file creation etc).

Reply:
Let's clean it up before merge, otherwise it might produce more conflicts for other branches rebases/merges